README.md

```diff
@@ -30,40 +23,41 @@ ocs3/
  |           +-- edu.gemini.seqexec.web.shared/
  |           +-- edu.gemini.seqexec.web.server/
  |           +-- edu.gemini.seqexec.web.client/
- +-- lib/
- +-- project/
+ +-- app/
 ```
-`lib` contains only unmanaged dependencies where `project` includes sbt definitions
+`app` contains build.sbt definitions for deployable applications
 
 ## Settings
 
 Settings and module versions are defined in the file `project/Settings.scala`
 
-## Dependencies
-
-Most dependencies are managed dependencies coming from either maven central or the Gemini repository.
-
-Intra-module dependencies are declared on `project/OcsBuild`. Being at the top level the project declarations can be used on each module's `build.sbt`
-
 ## Testing
 
-`ScalaTest` has been chosen as the test framework since it works in both scala.js and jvm. `ScalaCheck` is also supported running with `ScalaTest`. Sample tests using `scalatest` and `scalacheck` are included in some modules. You can run `test` at the root level or on a module basis.
+`ScalaTest` has been chosen as the test framework since it works in both scala.js and jvm. `ScalaCheck` is also supported running with `ScalaTest`.
+Sample tests using `scalatest` and `scalacheck` are included in some modules. You can run `test` at the root level or on a module basis.
 
 Note that IDEA doesn't support running tests on scala.js. See this [issue](https://github.com/scalatest/scalatest/issues/743)
 
+To tests the javascript side you need to install node and jsdom. In OSX using `brew` you can do:
+
+```
+brew install node
+npm install jsdom
```

Comment:
Thanks for checking this. the global install doesn't seem to work, you need to do it locally

Unfortunately `jsDependencies` won't help in this case. You use `jsDependencies`  to add js libraries to your program while `node` is the runtime and `jsdom` provides a `DOM` mockup. In the browser both are provided while we use `node` only to run the tests

I hope that when `Rhino` is phased away by scala.js they provide more foolproof instructions on installing `node`

I'll update the README a bit to clarify some of these concepts